resource/resource.go

```diff
@@ -19,6 +19,8 @@
 // and stylesheets.
 package resource
 
+//go:generate tsc -p ui/ts/
```

Comment:
Do we want to do this with `go generate`? I think that during development people would rather use go-bindata in debug mode and work on the typescript files without going though `go generate` and `go build`. We should have a command like `make js` to convert from ts to js, and then use `go generate` to convert from js to embedded.go.